src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/custom.py

```diff
@@ -265,8 +262,6 @@ def update_pool(client, pool_id, json_file=None, command_line=None,  # pylint:di
     except (ValidationError, ClientRequestError) as ex:
         raise CLIError(ex)
 
-update_pool.__doc__ = PoolUpdatePropertiesParameter.__doc__ + "\n" + StartTask.__doc__
```

Comment:
Out of curiosity, why concatenate two docstrings? Isn't the format of the new doc string broken?